runtime/runtime.go

```diff
@@ -45,6 +46,11 @@ var (
 	registeredPluginsMux sync.Mutex
 )
 
+const (
+	// default interval between OPA version report uploads
+	defaultUploadIntervalSec = int64(300)
```

Comment:
5 minutes feels a bit aggressive. I'd crank this up to an hour at least.

Reply:
Updated the interval to an hour.